examples/flask/flask_seguro/static/css/app.css

```diff
@@ -1,3 +1,7 @@
 body {
   padding-top: 50px;
+}
+
+.products{
```

Comment:
Needs a space before the `{`

Reply:
solved

Reply:
I meant a space between `.products` and the `{` like like that `.products {`. Sorry I wasn't clear